rpms/CentOS/6/x86_64/Dockerfile

```diff
@@ -1,10 +1,18 @@
 FROM centos:6
 # Install all the necessary tools to build the packages
 RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
-RUN yum -y install centos-release-scl gcc make git \
+RUN yum install -y centos-release-scl gcc make git \
     openssh-clients rpm-build sudo gnupg \
     automake autoconf libtool policycoreutils-python \
-    yum-utils epel-release redhat-rpm-config rpm-devel
+    yum-utils epel-release redhat-rpm-config rpm-devel \
+    autopoint gettext zlib zlib-devel nspr nspr-devel \
+    nss nss-devel kenel-headers magic magic-devel \
+    db4 db4-devel zlib zlib-devel rpm-build bison \
```

Comment:
Same here.

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/378/commits/90204700decee13e474b57e790795ab160dc1a52